const infiniti = true;

console.log(9);
const a = 5;

// const infinitPromise = new Promise((resolve, reject) => {
//   while (infiniti) {
//     resolve("Infiniti loop");
//   }
//   reject("Isn't Infiniti loop");
// });

// infinitPromise
//   .then((response) => {
//     console.log(response, "resp");
//   })
//   .catch((err) => {
//     console.log(err, "err");
//   });

console.log(Math.pow(a, 2));
console.log(99);
